Source/WebCore/Modules/webauthn/fido/AuthenticatorGetInfoResponse.cpp

```diff
@@ -134,7 +134,9 @@ Vector<uint8_t> encodeAsCBOR(const AuthenticatorGetInfoResponse& response)
     
     if (response.transports()) {
         auto transports = *response.transports();
-        deviceInfoMap.emplace(CBORValue(kCtapAuthenticatorGetInfoTransportsKey), toArrayValue(transports.map(WebCore::toString)));
+        deviceInfoMap.emplace(CBORValue(kCtapAuthenticatorGetInfoTransportsKey), toArrayValue(transports.map([](auto transport) {
+            return WebCore::toString(transport);
```

Comment:
Is this an intended part of your change? Since very WebAuthn-related, not WebRTC related! :-)